File: 1701.cc
#include "iostream"

int	main()
{
	std::string s;
	std::cin >> s;
	int res = 0, i = -1;
	while (++i < s.length())
	{
		int next = i + 1;
		if (i + 1 == s.length())
			next = 0;
		if (s[i] != s[next])
			continue ;
		res += s[i] - '0';
	}
	int res2 = 0, mid = s.length() / 2;
	i = -1;
	while (++i < s.length() / 2)
	{
		if (s[i] == s[i + mid])
			res2 += 2 * (s[i] - '0');
	}
	std::cout << res << '\n';
	std::cout << res2 << '\n';
}

File: 1701.py
s = open('_inputs_/1701.0').read().strip()
res = 0
for i in range(len(s)):
    R = i + 1
    if R == len(s):
        R = 0
    if s[i] != s[R]:
        continue
    res += int(s[i])
res2 = 0
mid = len(s) // 2
for i in range(mid):
    if s[i] == s[i + mid]:
        res2 += 2 * int(s[i])
print(res)
print(res2)
